### PR TITLE
ADAPT-4020, ADAPT-4021: Hide all trip content except hero and display alert after trip has started

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Hide all trip content except hero/ankle and display alert after trip has started

# Review By (Date)
- When does this need to be reviewed by?

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to https://deploy-preview-243--stanford-alumni.netlify.app/travel-study/trips/test-started-trip/
2. This trip has a start date set to June 2021, check that it's now displaying an alert at the top, and only the Hero + ankle are displayed like in the mockup
fae2a147-5d4c-48cc-8575-68cbdb61251b
3. Go to a trip with start date in 2022 https://deploy-preview-243--stanford-alumni.netlify.app/travel-study/trips/trip-preset-cruise
4. Check that all the trip content renders and there is no alert at the top

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/page-types/TripPage/TripPage.jsx [hunk 1]
```diff
@@ -140,6 +146,11 @@ const TripPage = (props) => {
 
   return (
     <SbEditable content={blok}>
+      {hasTripStarted && (
+        <Alert type="warning" label="Warning:" hasDismiss>
```
Comment (mattanglin) [inline src/components/page-types/TripPage/TripPage.jsx:150]: Do we want this to be dismissable?

Comment (yvonnetangsu) [reply src/components/page-types/TripPage/TripPage.jsx:150]: That's a good point. It's probably better to remove that option. I'll push up a tweak 😄

File: src/components/page-types/TripPage/TripPage.jsx [hunk 2]
```diff
@@ -140,6 +146,11 @@ const TripPage = (props) => {
...
+          <p className={styles.startAlert}>Too late...this trip has started.</p>
```
Comment (mattanglin) [inline src/components/page-types/TripPage/TripPage.jsx:151]: Is this the final copy for the Alert?

Comment (yvonnetangsu) [reply src/components/page-types/TripPage/TripPage.jsx:151]: Nope 😂  Jeanne is working on the final copy, but Julia asked me to build it using placeholder content first.

File: src/components/page-types/TripPage/TripPage.jsx (hunk 1 above)
Comment (yvonnetangsu) [reply src/components/page-types/TripPage/TripPage.jsx]: It's been updated. Looks like hasDismiss was set to true by default so I had to include the `hasDismiss={false}`

File: src/components/page-types/TripPage/TripPage.jsx (hunk 2 above)
Comment (mattanglin) [reply src/components/page-types/TripPage/TripPage.jsx:151]: Cool! That's what I figured, but thought maybe they decided to go loose on this one! 😂

Comment (yvonnetangsu) [reply src/components/page-types/TripPage/TripPage.jsx:151]: 🤣

**Review by mattanglin — APPROVED** [review]
Looks good to me! 👍🏼

**Pull request merged**